enacts/calc.py

```diff
@@ -121,6 +126,126 @@ def synthesize_enacts(variable, time_res):
     var_name = variable
     return xrds[var_name]
 
+
+def sql2GeoJSON(shapes_sql, db_sql):
```

Comment:
Do you have a justification for putting this shape stuff in calc rather than in maproom_utilities?

(Also the data reading stuff arguably doesn't belong in calc either.)